packages/databyss-ui/theming/buttons.js

```diff
@@ -74,6 +74,14 @@ const buttonVariants = {
     paddingTop: pxUnits(6),
     borderRadius: pxUnits(0),
   },
+  editSource: {
```

Comment:
- Let's call this something more generic, like `editAtomicInline`
- Border radius should use global from theme
- Use theme space values instead of px values